packages/vue/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@nx/vue",
+  "version": "0.0.1",
+  "private": true,
+  "description": "The Vue plugin for Nx contains executors and generators for managing Vue applications and libraries within an Nx workspace. It provides:\n\n\n- Integration with libraries such as Jest, Cypress, and Storybook.\n\n- Generators for applications, libraries, and more.\n\n- Library build support for publishing packages to npm or other registries.\n\n- Utilities for automatic workspace refactoring.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nrwl/nx.git",
+    "directory": "packages/vue"
+  },
+  "keywords": [
+    "Monorepo",
+    "Vue",
+    "Web",
+    "CLI"
+  ],
+  "main": "./index",
+  "typings": "./index.d.ts",
+  "author": "Victor Savkin",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "homepage": "https://nx.dev",
+  "schematics": "./generators.json",
+  "builders": "./executors.json",
+  "ng-update": {
```

I don't think we should use the Angular CLI compat syntax here, it won't ever be used in an Angular CLI workspace context, right?

```suggestion
  "nx-migrations": {
```